### DISCUSSION TITLE
Working Themes

### DISCUSSION THREAD

**Discussion by bradykelly** [jsonresume/jsonresume-theme-class#52] [Q&A]
I queried the npmjs registry for all packages with the prefix `jsonresume-theme` and got back a long list. Now I've made a little app that lets me select one and view my resume with that theme, but for more themes than not, I get a message that the theme is deprecated. Is there any development on themes besides this one, is there maybe a list of themes that are still active? Is there some way better than hijacking your repo's discussions to ask questions like this?

**Comment by SethFalco** [comment]
Sorry for the slow response here! Not sure why I didn't get notified of this!

> Is there any development on themes besides this one

As far as first-party or officially supported themes go, I think this may be the only one, unfortunately.

I have been meaning to find another theme candidate to either contact the original maintainer and try to collaborate, or otherwise fork it and bring it under our org, but have been too busy to initiate this. (Typical, I know. ^-^')

> is there maybe a list of themes that are still active

We don't officially maintain a list. However, I have been interested in building a tool like [tldr-pages/tldr-pages-test-hardness](https://github.com/tldr-pages/tldr-pages-test-harness) but for JSON Resume so we can run automated checks on themes to see what features they support and if it's up-to-date with the spec. This may help with curating such a list.

---

Theme support has been a long-standing challenge with JSON Resume, as most people (understandably so) will only maintain their theme for as long as they themselves are using it and actively applying for work. Even among most of the themes in the org, as soon as the maintainer of the theme has landed a job, maintenance also drops.

While the standard/spec itself is stable, the ecosystem itself isn't.
